src/main/java/com/aws/greengrass/componentmanager/plugins/docker/EcrAccessor.java

```diff
@@ -57,7 +57,8 @@ public EcrClient getClient(String region) {
             region = Coerce.toString(deviceConfiguration.getAWSRegion());
         }
 
-        return EcrClient.builder().httpClient(ProxyUtils.getSdkHttpClient())
```

Comment:
Let's mark `getSdkHttpClient` as deprecated and any public method that returns a client and not a builder.

Reply:
updated